x-pack/plugins/security_solution/common/api/entity_analytics/risk_engine/common.schema.yaml

```
@@ -32,10 +32,7 @@ components:
       type: string
 
     Filter:
```

Let's add a very permissive type here.
```suggestion
    Filter:
      type: object
```